portfolio/src/main/webapp/script.js

```diff
@@ -13,19 +13,19 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Displays one of my favorite foods
  */
-function addRandomGreeting() {
-  const greetings =
+function addRandomFood() {
```

Comment:
I'm ok with this staying in unused for now and going in with this PR, but in the future it would be good practice to delete this function and add it back if / once there is html using it. For now, there's no enabled way to verify that this function works.

Stuff like this can happen a lot and we end up with lots of things that are unused in the codebase. We add them with the intent of using them later, forget about them and never delete them.